spec/features/ncr_work_orders_spec.rb

```diff
@@ -108,6 +108,15 @@
         end
       end
 
+      it "shows hint text for amount field", js: true do
+        visit '/ncr/work_orders/new'
+        page.execute_script "document.getElementById('ncr_work_order_amount').scrollIntoView()"
```

Comment:
WOOF! hate it when we have to do this type of thing in feature specs, but I know it is a necessary evil at times.

To make it more obvious what these two lines are doing, what about putting 113 and 114 into a private method called something like `hover_over_amount_field_popover` ?